module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -738,4 +752,15 @@ public function getCheckboxFacets(
         // Return modified list:
         return $facets;
     }
+
+    /**
+     * Return virtual field name for custom filters
+     *
+     * @return string
+     */
+    protected function getCustomFilterFieldName(): string
```

Comment:
Is there a meaningful benefit to adding a method for this, vs. setting a property in the constructor? I don't mind this approach, but the property might be simpler.

Reply:
No, and the property set in constructor avoids doing it repeatedly, so a much better approach. Changed!